calculator.rb

```diff
@@ -0,0 +1,49 @@
+class Calculator
+    def initialize
```

Comment:
```
calculator = Calculator.new(10, 15, 20)
calculator.add
calculator.subtract
```

@Joydeep-ruby , here the method should be like this and there should not be any restriction on the number of arguments in the case of addition and multiplication.

Another point, there should be a line gap when the method is being defined.